src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InitialTestProcessTests.cs

```diff
@@ -20,21 +21,33 @@ public void InitialTestProcess_ShouldThrowExceptionOnFail()
         {
             var testRunnerMock = new Mock<ITestRunner>(MockBehavior.Strict);
             testRunnerMock.Setup(x => x.RunAll(It.IsAny<int?>(), null)).Returns(new TestRunResult { Success = false });
-            testRunnerMock.Setup(x => x.CaptureCoverage())
+            testRunnerMock.Setup(x => x.CaptureCoverage(false, false))
                 .Returns(new TestRunResult { Success = false });
-            testRunnerMock.Setup(x => x.DiscoverNumberOfTests()).Returns(0);
+            testRunnerMock.Setup(x => x.DiscoverNumberOfTests()).Returns(1);
 
             var exception = Assert.Throws<StrykerInputException>(() => _target.InitialTest(testRunnerMock.Object));
         }
 
+        [Fact]
+        public void InitialTestProcess_ShouldThrowWhenNoTest()
```

Comment:
Why did you add this? I think it is fine if we keep running stryker on a project with 0 tests.